### ISSUE TITLE
eclipse launch script created for MacOS is buggy

### ISSUE THREAD

**Issue by hohwille** [devonfw/devon-ide#204]
```
➜  devon devon
devon-ide has environment variables have been set for /projects/devon (workspace main)
➜  devon devon eclipse add-plugin cobigen
Installing eclipse plugin com.devonfw.cobigen.eclipse.feature from https://dl.bintray.com/devonfw/cobigen.p2/
/projects/devon/software/eclipse/eclipse: line 2: /projects/ide-test/software/eclipse/Eclipse.app/Contents/MacOS/eclipse: No such file or directory

******** ATTENTION ********
WARNING: Installation of plugin com.devonfw.cobigen.eclipse.feature failed. Please try to install manually.
```

As you can see above I am in `devon` IDE but for some reason devon command resolves to `ide-test` IDE. Not a clue where this comes from.
```
$ cat ~/.devon/home
DEVON_IDE_HOME=/projects/devon
```

**Comment by hohwille** [comment]
```
echo $ECLIPSE_HOME

```

**Comment by hohwille** [comment]
```
$ env | grep test
```
No output.

**Comment by hohwille** [comment]
Aha, cool. Got it. This is from #152.
On MacOS a eclipse command is missing in Eclipse installation and therefore as a workaround we create one. This is generated with the IDE installation specific path (`${DEVON_IDE_HOME}/software/eclipse`). However, if `DEVON_SOFTWARE_PATH` is defined it actually needs to point to the physical Eclipse installation or may not use an absolute path.

**Comment by hohwille** [comment]
Fixed with PR #205
